internal/mountinfo/mountinfo_linux_test.go

```diff
@@ -0,0 +1,26 @@
+package mountinfo
+
+import (
+	"os"
+	"testing"
+
+	"github.com/sourcegraph/log/logtest"
+)
+
+func Test_MountInfo_SmokeTest_Github_Actions(t *testing.T) {
+	if os.Getenv("GITHUB_ACTIONS") == "" {
+		t.Skip("this smoke test should only run in our Github Actions CI environment")
+	}
+
+	logger := logtest.Scoped(t)
+
+	// A simple smoke test to verify that we can find the storage device
+	// for the location of the zoekt checkout on the CI agent.
+	filePath := os.Getenv("GITHUB_WORKSPACE")
```

Comment:
go runs tests in the same directory as the source code, so instead of making this github specific you can just do cwd. Then this test should just work on any linux env?